projects/spectator/src/lib/internals.ts

```diff
@@ -54,7 +47,7 @@ export class Spectator<C> {
    * Run detect changes on the host component
```

Comment:
hmm I should also update the comment here.

@NetanelBasal do you remember why it was done this way? Is there a time when we need to run change detection for host when not having the host?

Reply:
✅ comment fixed.